index.html

```diff
@@ -68,6 +68,14 @@
       </p>
     </section>
     <section id="sotd">
+      <p>
+        The Media &amp; Entertainment Interest Group plans advance the work on
```

Comment:
Is this a grammar typo and should it be "plans _to_ advance"? (I think so, just checking). If so, also consider using "intends to" in place of "plans to" unless there's an actual y'know, "plan".